node/src/chain_spec.rs

```diff
@@ -469,9 +468,8 @@ fn lerna_genesis(
 			key: root_key,
 		},
 		asset_registry: AssetRegistryConfig {
-			core_asset_id: CORE_ASSET_ID,
-			asset_ids: vec![],
-			next_asset_id: 1,
+			asset_names: vec![],
+			native_asset_name: b"HDX".to_vec(),
```

Comment:
@enthusiastmartin shouldn't we do some storage migration because of this? i guess the empty array is fine, but the native asset name won't be set in the live chain. alternatively, we could sudo set the storage value manually.

Reply:
did you test runtime upgrade?

Reply:
Hm, correct - native details wont be in the storage. 

It is actually not an issue. We (so far) dont use the native name at all nor we need the native details in the storage. We always use only the native asset id.

And we can do it manually anyway.

However, I'll look into the upgrade (so i learn one or thing as well )